### PR TITLE
IronLazyPagesBehavior should include IronSelectableBehavior

### LINKED ISSUE #28: Incorrect urls in demos
### Description

Some of the urls in the demos are broken.

https://github.com/TimvdLippe/iron-lazy-pages/blob/master/demo/index.html#L47
https://github.com/TimvdLippe/iron-lazy-pages/blob/master/demo/index.html#L139

### LINKED ISSUE #29: Move IronSelectableBehavior into IronLazyPagesBehavior
### Description

The IronSelectableBehavior should be a dependency inside of IronLazyPagesBehavior since it depends on it.

### PR DESCRIPTION
**Pull request by cecilia-sanare**
- IronSelectableBehavior is now a dependency of IronLazyPagesBehavior
- Fixed issues with various demos pulling in the wrong html file

this closes #28 and closes #29

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -136,10 +136,10 @@ <h4 class="centered">All pages animated the same way</h4>
               loading="{{loading}}"
               entry-animation="fade-in-animation"
               exit-animation="fade-out-animation">
-            <template is="iron-lazy-page" data-route="foo" path="element.html">
```
Comment (TimvdLippe) [inline demo/index.html:139]: Please revert this change. It is intended to test if nonexisting imports keep on working

File: iron-lazy-pages-behavior.html [hunk 2]
```diff
@@ -1,104 +1,111 @@
...
-    // as the selected page is the only one visible, activateEvent
-    // is both non-sensical and problematic; e.g. in cases where a user
-    // handler attempts to change the page and the activateEvent
-    // handler immediately changes it back
-    // Disabled for this element.
-    activateEvent: {
-      type: String,
-      value: null
-    },
+  Polymer.IronLazyPagesBehaviorImpl = {
```
Comment (TimvdLippe) [inline iron-lazy-pages-behavior.html]: Please revert the indentation changes

**Review by TimvdLippe — CHANGES_REQUESTED** [review]

File: demo/index.html (hunk 1 above)
Comment (cecilia-sanare) [reply demo/index.html:139]: Done!

File: iron-lazy-pages-behavior.html (hunk 2 above)
Comment (cecilia-sanare) [reply iron-lazy-pages-behavior.html]: Done!

File: demo/index.html [hunk 3]
```diff
@@ -139,7 +139,7 @@ <h4 class="centered">All pages animated the same way</h4>
             <template is="iron-lazy-page" data-route="foo" path="element.html">
               <span>Foo page</span>
             </template>
-            <template is="iron-lazy-page" data-route="bar" path="x-bar.html">
```
Comment (TimvdLippe) [inline demo/index.html:142]: This one should be reverted too actually :smile:

File: iron-lazy-pages-behavior.html [hunk 4]
```diff
@@ -1,5 +1,8 @@
+<link rel="import" href="../iron-selector/iron-selectable.html">
+
 <script>
-Polymer.IronLazyPagesBehavior = {
+  /** @polymerBehavior Polymer.IronLazyPagesBehavior */
```
Comment (TimvdLippe) [inline iron-lazy-pages-behavior.html]: Small indentation nit. Also TIL about the necessity for `@polymerBehavior`

**Review by TimvdLippe — CHANGES_REQUESTED** [review]
LGTM with 2 more small nits.

File: demo/index.html (hunk 3 above)
Comment (cecilia-sanare) [reply demo/index.html:142]: Done!

File: iron-lazy-pages-behavior.html (hunk 4 above)
Comment (cecilia-sanare) [reply iron-lazy-pages-behavior.html]: Not sure how I missed that!

Should be good now!

**Review by TimvdLippe — APPROVED** [review]

**Comment by TimvdLippe** [comment]
Awesome, thanks a lot! :tada:
